Jenkinsfile

```diff
@@ -79,15 +80,15 @@ pipeline {
         stage('Build') {
             steps {
                 echo 'Building'
-                sh 'mvn -U -B -e clean install -DskipTests'
+                sh 'mvn -U -B -e clean install -DskipTests -T1C'
             }
         }
 
         stage('Tests') {
             steps {
                 echo 'Running tests'
                 // all tests is very very long (10 hours on Apache Jenkins)
-                sh 'mvn -B -e test'
+                sh 'mvn -B -e test -T1C'
```

Comment:
That's probably ambitious for test phase.

Reply:
Agree. As isolation is poor we may try to leverage Jenkins plugins to split tests between different runners to keep good isolation level.
Let's first look at the result of this try (:

Reply:
Also we might blow up the test runner as parallel runs are achieved at the maven module level.

Of course, I'd be happy to not rely on surefire forks and rely on maven concurrency instead.

Thanks for your work on that with @jeantil BTW.